### PR TITLE
Make pairs stored more optimized on Android

### PR DESCRIPTION
**Pull request by osdnk**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -28,7 +28,6 @@
     "start:clean": "watchman watch-del-all && rm -rf $TMPDIR/react-* && rm -rf $TMPDIR/metro-* && rm -rf $TMPDIR/haste-map-* && react-native start -- --reset-cache",
     "test": "jest test.js",
     "update-env": "curl -H 'Authorization: token '$GITHUB_TOKEN -H 'Accept: application/vnd.github.v3.raw' -O  -L https://api.github.com/repos/rainbow-me/rainbow-env/contents/dotenv && rm -f .env && mv dotenv .env",
-    "ganache": "source .env && ganache-cli --f https://mainnet.infura.io/v3/${INFURA_PROJECT_ID_DEV} -p 7545 -i 1 -b 10",
```
Comment (jinchung) [inline package.json:31]: did you mean to remove this?

Comment (osdnk) [reply package.json:31]: No, im sorry

**Comment by jinchung** [comment]
Do you mind if you time how long it takes to execute the getAllTokens function with and without the compression logic? I'm just curious how much longer it takes given that it'll be a lot of tokens (with multiple properties to compress for each), and it isn't obvious to me how the savings in space vs running the overhead to execute the compression balance out.

**Comment by osdnk** [comment]
I see that it doesn't make the app faster. Waiting for @skylarbarrera's confirmation

**Comment by skylarbarrera** [comment]
also not seeing much difference

**Pull request closed without merging**